src/platform/packages/shared/response-ops/rule_form/src/rule_page/rule_page_confirm_create_rule.test.tsx

```diff
@@ -1,49 +0,0 @@
-/*
```

Comment:
I'm not taking the decision to remove a test file lightly, but take a look and consider this. We're testing to see if `<EuiConfirmModal>` renders its children, and if clicking the buttons execute the correct functions.

These functions are already covered in the [EUI package's test file for EuiConfirmModal](https://github.com/elastic/eui/blob/main/packages/eui/src/components/modal/confirm_modal.test.tsx) and I don't think we should be duplicating this just to say we have test coverage on every component.